test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14177,7 +14177,7 @@ def _unix_initd(self, hostname, op, conf_file, init_script, daemon):
             elif daemon == 'comm' and conf.get('PBS_START_COMM', 0) != 0:
                 dconf['PBS_START_COMM'] = 1
             for k, v in dconf.items():
-                init_cmd += ["%s=" % k + str(v)]
+                init_cmd += ["%s=%s" % (k, str(v))]
```

Comment:
Could you explain this change?  Won't this turn 'PBS_START_COM=1' into 'PBS_START_COM1=' ?

Reply:
Hi Bhroam, Here I am passing two string variables to "%s=%s" , hence I have put them inside a tuple.  As dconf is the dictionary, key will be PBS_START_COM and value will be 1 and the string will be PBS_START_COM=1.

Reply:
@bhroam you are right, before last commit it was happening to change 'PBS_START_COM=1' into 'PBS_START_COM1=' but as you can see ```k + str(v)``` is replaced with `k, str(v)` in latest commit so it is fixed...

Reply:
Oh true.  I must have been looking at what was removed.  Yeah, the new code looks correct.  I sign off.